isUnique/isUnique.js

```diff
@@ -0,0 +1,27 @@
+/* 
+ * Implement an algorithm to determine if a string has all unique characters.
+ * Extra Credit - Answer this question - What if you cannot use additional data structures?
+ */
+// create new variable 'splitString' that will hold string.split()
+// split the string
+// iterate through stringSplit array
+// if strSplit[i] === indivi
+
+const isUnique = (str) => {
+  let count = 0;
+  const splitString = str.split('');
```

Comment:
One thing that is cool about strings, is you don't even need to split them to loop over them 👍